Azure Monitor Exporter - Update readme (#17094)
Azure/azure-sdk-for-net: sdk/monitor/Microsoft.OpenTelemetry.Exporter.AzureMonitor/README.md

## Contributing

See [CONTRIBUTING.md](https://github.com/Azure/azure-sdk-for-net/blob/master/CONTRIBUTING.md) for details on contribution process.

## Release Schedule

This exporter is under active development.

> The library is not yet _generally available_, and releases aren't guaranteed to conform to a specific version of the specification. Future releases will not attempt to maintain backwards compatibility with previous releases. Each alpha and beta release includes significant changes to the API and SDK packages, making them incompatible with each other.

Reviewer: ```suggestion
The library is not yet _generally available_, and is not officially supported. Future releases will not attempt to maintain backwards compatibility with previous releases. Each beta release includes significant changes to the exporter package, making them incompatible with each other.
```
